src/v2/guide/syntax.md

```diff
@@ -82,23 +82,23 @@ Ces expressions seront évaluées en tant que JavaScript au sein de la portée d
 
 ## Directives
 
-Les directives sont des attributs spéciaux avec le prefixe `v-`. Les valeurs attendues pour les attributs de directives sont **une unique expression JavaScript** (A l'exception de `v-for`, qui sera discuté plus loin). Le travail d'une directive est d'appliquer réactivement des effets secondaires au DOM quand la valeur de son expression change. Revenons à l'exemple vu dans l'introduction :
+Les directives sont des attributs spéciaux avec le prefixe `v-`. Les valeurs attendues pour les attributs de directives sont **une unique expression JavaScript** (A l'exception de `v-for`, qui sera expliquée plus loin). Le travail d'une directive est d'appliquer réactivement des effets secondaires au DOM quand la valeur de son expression change. Revenons à l'exemple vu dans l'introduction :
 
 ``` html
 <p v-if="seen">Maintenant vous me voyez</p>
 ```
 
-Ici, la directive `v-if` retirerait / insererait l'élement `<p>` en se basant sur l'évaluation à vrai de la valeur de l'expression `seen`.
```

Comment:
Ici nous avons traduit 

> determined by the truthiness of the data property isActive.

par

> présente si la propriété isActive est évaluée à vrai.

car ce sont les termes utilisés dans cette explication :  https://github.com/vuejs-fr/vuejs.org/blob/master/src/v2/guide/class-and-style.md#liaison-de-classes-html

Pour moi la traduction est plus que correcte ! On peut garder ainsi non ?